<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTrilho01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow;
border: 3px dashed black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho03">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow;
border: 3px dashed black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho02">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange;
border: 3px dashed black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho04">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>280</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border: 3px solid black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho06">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>210</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow;
border: 3px dashed black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrilho05">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>210</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange;
border: 3px dashed black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem01">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  blue;
color:white;</string>
    </property>
    <property name="text">
     <string>01</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem02">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  brown;
color:white;</string>
    </property>
    <property name="text">
     <string>02</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem03">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  purple;
color:white;</string>
    </property>
    <property name="text">
     <string>03</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem04">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  darkblue;
color:white;</string>
    </property>
    <property name="text">
     <string>04</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem06">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  green;
color:white;</string>
    </property>
    <property name="text">
     <string>06</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem05">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  darkgreen;
color:white;</string>
    </property>
    <property name="text">
     <string>05</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
